### ISSUE TITLE
U+1168E TAKRI LETTER NGA overlaps post-base components

### ISSUE THREAD

**Issue by dscorbett** [googlefonts/noto-fonts#1694]
### Font
NotoSansTakri-Regular.otf

### Where the font came from, and when
Site: https://github.com/googlefonts/noto-fonts/blob/115d38430d957d38307457c036302b7bdbe0bbc4/phaseIII_only/unhinted/otf/NotoSansTakri/NotoSansTakri-Regular.otf
Date: 2020-04-04

### Font version
Version 2.001

### Issue
U+1168E TAKRI LETTER NGA overlaps post-base components.

### Character data
𑚎𑚯𑚎𑚶𑚩
U+1168E TAKRI LETTER NGA
U+116AF TAKRI VOWEL SIGN II
U+1168E TAKRI LETTER NGA
U+116B6 TAKRI SIGN VIRAMA
U+116A9 TAKRI LETTER HA

### Screenshot
![𑚎𑚯𑚎𑚶𑚩](https://user-images.githubusercontent.com/1124347/78510877-64d0da80-7766-11ea-9f40-3ac22e8bb0de.png)

**Comment by nik9hil** [comment]
Hey, I have been trying to type using this script in MS Word 2016, Windows 10. Whenever I try to type in it, it either reverts back to some other English font or simply ends up typing in English. I tried copying the takri text to Word, that you have put here, but they appeared tofu until I converted them deliberately to Noto Sans Takri Regular. But then I am still not able to edit it. 
Being native speaker, I can tell that the issue you have raised here can cause problem for some words.
